programs/README.md

```diff
@@ -233,7 +236,7 @@ The below table illustrates this on the [Silesia compression corpus].
 | `zstd -10 --long`| `3.566` | `16.2 MB/s`       | `415.7 MB/s`   |
 
 
-#### zstdgrep
+### zstdgrep
```

Comment:
Unrelated: I see below that we mention the limitations of zstdgrep. Guess we should also mention the grep options that we don't support currently? I can put up a pr soon

Reply:
Sure !